dist/Views/scstormrunoff.html

```diff
@@ -0,0 +1,141 @@
+                                                </tr>
+                                                <tr>
+                                                    <td>Main Channel Length (miles):</td>
+                                                    <td><b>{{vm.mainChannelLength}}</b></td>
+                                                </tr>
+                                                <tr>
+                                                    <td>Main Channel Slope (feet per mile):</td>
+                                                    <td><b>{{vm.mainChannelSlope}}</b></td>
+                                                </tr>
+                                                <tr>
+                                                    <td>Total impervious area (%):</td>
+                                                    <td><b>{{vm.totalImperviousArea}}</b></td>
+                                                </tr>
+                                            </tbody>
+                                        </table>
+                                        <h3>Summary</h3>
+                                        <table id="BohmanUrbanSummaryTable" class="table table-condensed smalltext">
+                                            <tbody>
+                                                <tr>
+                                                    <td>Weighted Runoff Volume:</td>
+                                                    <td><b>{{vm.ReportData.BohmanUrban1992.WeightedRunoff}}</b></td>
+                                                </tr>
+                                            </tbody>
+                                        </table>
+        
+                                        <div style='width:650px; margin:auto;' class="page_break" id="legend">
+                                            <nvd3 options='vm.ReportOptions' data='vm.ReportData.BohmanUrban1992.Graph'></nvd3>
+                                        </div>
+        
+                                        <h3>Tabular Hydrograph</h3>
+                                        <table id="BohmanUrbanHydrograph" class="table table-condensed" style="page-break-inside:avoid; font-size: 11px;">
+                                            <thead>
+                                                <tr>
+                                                    <th>Time</th>
+                                                    <th>Q</th>
+                                                </tr>
+                                            </thead>
+                                            <tbody>
+                                                <tr ng-repeat="item in vm.reportData.time_coordinates">
+                                                    <td>{{item}}</td>
+                                                    <td>{{vm.reportData.discharge_coordinates[$index]}}</td>
```

Comment:
Reduce the number of decimal places to 1 for the Q values. Then we won't have values like 296.09999999999997 (should be 296.1)